worker/worker.go

```diff
@@ -196,6 +203,42 @@ func (w *Worker) EmitEvent(
 		args...)
 }
 
+// cancel implements com.redhat.Yggdrasil1.Worker1.Cancel method by calling the
+// worker's cancelRxFunc in a goroutine.
+func (w *Worker) cancel(addr string, id string, cancelID string) *dbus.Error {
+	// If worker doesn't implements the cancelation function it does nothing
```

Comment:
I can see small grammar mistakes ... there should be: "doesn't implement cancellation".